sqswatcher/plugins/slurm.py

```diff
@@ -83,7 +85,8 @@ def _restart_compute_node_worker(args):
     try:
         _restart_compute_daemons(*args)
         return hostname, True
-    except Exception:
+    except Exception as e:
+        log.error("Failed when restarting compute node %s with error %s", hostname, e)
```

Comment:
I see this as a duplication

Reply:
removed the log from _restart_compute_daemons